package-lock.json

```diff
@@ -201,7 +201,6 @@
       "resolved": "https://registry.npmjs.org/combined-stream/-/combined-stream-1.0.7.tgz",
       "integrity": "sha512-brWl9y6vOB1xYPZcpZde3N9zDByXTosAeMDo4p1wzo6UMOX4vumB+TP1RZ76sfE6Md68Q0NJSrE/gbezd4Ul+w==",
       "dev": true,
-      "optional": true,
```

Comment:
These changes look unrelated here. I last generated this file using npm 6.7.0, maybe a older version was used here.

Reply:
Ah possibly - I was confused when that appeared in the merge.